packages/notebook/src/widget.ts

```diff
@@ -1189,6 +1197,7 @@ export namespace StaticNotebook {
      * - 'none': Attach all cells to the viewport
      */
     windowingMode: 'defer' | 'full' | 'none';
+    accessKernelHistory: boolean;
```

Comment:
Could you add a doc string for this interface?

Reply:
And it should be optional as it is changing the API for `set notebookConfig`. This probably requires to modify that setter to use the default value if it is not provided.

```suggestion
    accessKernelHistory?: boolean;
```